src/CarbonAware/test/Configuration/DataSourcesConfigurationTests.cs

```diff
@@ -57,4 +57,37 @@ public void AssertValid_DataSourceNotInConfigurationSection_ThrowsException(stri
         Assert.That(ex!.Message, Contains.Substring(errorMessage));
     }
 
+    [Test]
```

Comment:
This is great test. It feels like this is another candidate to be a `TestCase` method.